Mathlib/FieldTheory/NormalClosure.lean

```diff
@@ -282,4 +282,24 @@ lemma normal_iff_forall_map_eq : Normal F K ↔ ∀ σ : L →ₐ[F] L, K.map σ
 lemma normal_iff_forall_map_eq' : Normal F K ↔ ∀ σ : L ≃ₐ[F] L, K.map ↑σ = K :=
 ⟨fun h σ ↦ normal_iff_forall_map_eq.1 h σ, fun h ↦ normal_iff_forall_map_le'.2 (fun σ ↦ (h σ).le)⟩
 
+instance instSMulMemClass : SMulMemClass (IntermediateField F L) F L :=
+  ⟨fun _ _ hx ↦ smul_mem _ hx⟩
+
+@[simp]
```

Comment:
This proof should probably be in tactic mode rather than in term mode

Reply:
I'll take some time to figure out, but now I'm outside so I'll work on it later.

Reply:
Although it is in #16993, do you mean like this?
` ` ` lean
instance instSMulMemClass : SMulMemClass (IntermediateField k K) k K where
  smul_mem := by
    intro _ _ _ hx
    exact IntermediateField.smul_mem _ hx
` ` `

Reply:
I would be interested to hear @faenuccio's reasoning here.

Reply:
No, I am sorry, I was not speaking about the `instance instSMulMemClass` (whose term-mode proof is perfectly fine), I was speaking about `lemma normalClosure_map` whose proof is long-ish, morally written in tactic mode, but presented in term mode.

Reply:
Oh I think it's me who suggested it and I still prefer it in term mode. (The only difference in tactic mode would be a `by` and an `exact`), but of course this is only aesthetics.

Reply:
No, not only: it only permits to "navigate" the proof if someone is reading the code. So the "by" and the "exact" in my opinion do a good job. I prefer term mode if the term is self-explanatory, but if one needs to "understand" it, I very much prefer tactic mode. I agree it is the author's choice, at the end, though.